src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlBuffer.cs

```diff
@@ -42,55 +41,67 @@ internal enum StorageType
         internal struct DateTimeInfo
         {
             // This is used to store DateTime
-            internal Int32 daypart;
-            internal Int32 timepart;
+            internal int daypart;
```

Comment:
Could you exclude this file changes as part of this PR?
I don't mind the changes, but not necessarily here.

Reply:
I also wanted to mention.. there's a more changes than this required to make types uniform. I've seen this before but I believe a separate PR to make all primitive type changes would be better.

Reply:
There are many other files that need primitive changes like this but I chose to try and keep this PR small so I stopped once I reached 30 changes. I'll revert this file an put it in another batch.